src/main/java/org/usfirst/frc4904/robot/commands/controlpanel/SpinToColor.java

```diff
@@ -0,0 +1,36 @@
+package org.usfirst.frc4904.robot.commands.controlpanel;
+
+import org.usfirst.frc4904.standard.commands.RunUntil;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+import java.util.function.Supplier;
+
+import org.usfirst.frc4904.robot.commands.controlpanel.StartPanelMotor;
+import org.usfirst.frc4904.robot.commands.controlpanel.StopPanelMotor;
+
+public class SpinToColor extends ColorSensor {
+    private final Motor motor;
+    private final Color color;
+
+    /**
+     * @param motor The motor to spin the control panel.
+     * @param color A string representation of the color that needs to be spinned to
+     */
+    public SpinToColor(Motor motor, ColorSensor.Color color) {
+        super();
+        this.motor = motor;
+        this.color = color;
+
+        addRequirements(this.motor);
+
+        Supplier<Boolean> isDone = () -> {
+            Color colorResult = super.getColor();
+            if (colorResult == null) {
+                return false;
+            }
+            return colorResult == this.color;
+        };
+
```

Comment:
Similar to the comment in the previous script, I am not 100% sure if line 34 is how it is supposed to be done.

Reply:
I'd also recommend taking in a `ColorSensor` here.